### PR TITLE
Add note on JDBC libs for lookups

### PR DESCRIPTION
**Pull request by jnaous**
Add documentation on needed lib files to make JDBC lookups work.

### CODE REVIEW

File: docs/development/extensions-core/lookups-cached-global.md [hunk 1]
```diff
@@ -368,6 +368,12 @@ The JDBC lookups will poll a database to populate its local cache. If the `tsCol
 }
 ```
 
+> If using JDBC, you will need to add your database's client JAR files to the extension's directory.
+> For MySQL, you can get it from https://dev.mysql.com/downloads/connector/j/, and for Postgres, from
+> https://jdbc.postgresql.org/download.html or from the `extensions/postgresql-metadata-storage/`.
```
Comment (ccaominh) [inline docs/development/extensions-core/lookups-cached-global.md]: Typo: from the `extensions/postgresql-metadata-storage/` -> the `extensions/postgresql-metadata-storage/` directory.

Or: from the `extensions/postgresql-metadata-storage/` -> from `extensions/postgresql-metadata-storage/`

File: docs/development/extensions-core/lookups-cached-global.md [hunk 2]
```diff
@@ -368,6 +368,12 @@ The JDBC lookups will poll a database to populate its local cache. If the `tsCol
...
+> Copy or symlink the downloaded file to
+> `extensions/lookups-cached-global` under the distribution root directory.
```
Comment (ccaominh) [inline docs/development/extensions-core/lookups-cached-global.md]: I believe the directory should be `extensions/druid-lookups-cached-global`.

File: docs/development/extensions-core/lookups-cached-global.md (hunk 1 above)
Comment (jnaous) [reply docs/development/extensions-core/lookups-cached-global.md]: Fixed

File: docs/development/extensions-core/lookups-cached-global.md (hunk 2 above)
Comment (jnaous) [reply docs/development/extensions-core/lookups-cached-global.md]: Fixed

**Comment by gianm** [comment]
TC is failing, but it's because of the thrift jar issue. I won't bother merging master into this branch, since this change is docs-only.

**Pull request merged**